lk2nd/smp/cpu-boot.c

```diff
@@ -44,26 +44,135 @@ static inline uint32_t read_mpidr(void)
 	return BITS(res, 23, 0);
 }
 
-static uint32_t read_phandle_reg(const void *dtb, int node, const char *prop)
+/**
+ * @brief This function used to read value by index of specified property
```

Comment:
We mostly use [Kerneldoc](https://docs.kernel.org/doc-guide/kernel-doc.html) for annotations in lk2nd codebase.

FWIW my "policy" is that only "global" functions must be documented (that is, ones that are in the global include headers) as I try to maintain a cross-referencer at https://lxr.trvn.ru that makes use of those kerneldocs. On the other hand, documentation for generic functions like this is surely appreciated